resources/lib/unpause_jumpback.py

```
@@ -335,7 +345,83 @@ def onAVStarted(self):
+        try:
+            current_time = self.getTime()
+
+            # Don't jump back if we're near the beginning
+            if current_time <= self.jump_back_secs_after_seek:
+                Logger.info(f'Current time {current_time}s too close to start, skipping seek jumpback')
+                return
+
+            # Throttle: ignore rapid successive seeks (within 1 second)
+            current_timestamp = time.time()
+            if self.last_seek_time > 0 and (current_timestamp - self.last_seek_time) < 1.0:
+                Logger.info(f'Ignoring rapid seek (within 1 second of last seek)')
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)